src/vanilla.ts

```diff
@@ -1,6 +1,18 @@
+/**
+ * @deprecated `State` is renamed to `UnknownState`,
+ * `State` will be removed in next major
```

Comment:
I would wanted to drop `State` entirely, and not even export `UnknownState`.
But, this will probably be too big change, so let's keep it.

I would even want to make it `unknown`. If it were possible, we could consider dropping.